mmtrack/models/reid/linear_reid_head.py

```diff
@@ -32,17 +38,38 @@ def __init__(self,
                  norm_cfg=None,
                  act_cfg=None,
                  num_classes=None,
-                 loss=dict(type='CrossEntropyLoss', loss_weight=1.0),
+                 loss_cls=dict(type='CrossEntropyLoss', loss_weight=1.0),
+                 loss_triplet=dict(
+                     type='TripletLoss', margin=0.3, loss_weight=1.0),
+                 cal_acc=False,
                  topk=(1, )):
-        super(LinearReIDHead, self).__init__(loss=loss, topk=topk)
+        super(LinearReIDHead, self).__init__(
```

Comment:
1. If you want to do that, there should be `loss=loss_cls` in the init function, not the hard code. 
1. We can keep the original `loss` in the arguments, and add `loss_pairwise` for triplet loss